### ISSUE TITLE
Forbid assignment to a single unused variable

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1032]
# Rule request

## Thesis

Flake8 is ok with unused variables when they start from underscore. However, sometimes they are unnecessary, and flake8 won't warn about it.

```python
# bad
_ = something()
_, _ = something()
# good
something()

# bad
a, b, _, _ = something()
# good
a, b, *_ = something()
```

## Reasoning

All unnecessary code must be EXTERMINATED.

**Comment by sobolevn** [comment]
Already done! Thanks!

```
(.venv) ~/Documents/github/wemake-python-styleguide  master ✔                     1 ⚠️
» echo '_ = 1' > text.py

(.venv) ~/Documents/github/wemake-python-styleguide  master ✗                        
» flake8 text.py 

text.py

  1:1      C101  Coding magic comment not found
  _ = 1
  ^

  1:1      WPS122 Found all unused variables definition: _
  _ = 1
  ^

Full list of violations and explanations:
https://wemake-python-stylegui.de/en/0.12.5/pages/usage/violations/

```

**Comment by orsinium** [comment]
Ah, something wrong with my environment, sorry. Great.
